utils/tests/test_enums.py

```
@@ -0,0 +1,810 @@
+        # UI themes should be standard
+        theme_names = [theme.name for theme in UITheme]
+        assert "LIGHT" in theme_names
+        assert "DARK" in theme_names
+
+    def test_enum_values_are_consistent(self):
+        """Test that enum values follow consistent patterns."""
+        # String enums should use lowercase values
+        for theme in UITheme:
+            assert theme.value.islower()
+
+        for level in LogLevel:
```

Ruff (0.12.2)

Loop control variable `level` not used within loop body

Rename unused `level` to `_level`

(B007)